app/controllers/users/confirmations_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Users::ConfirmationsController < Devise::ConfirmationsController
```

Comment:
- [ ] It would be nice if you fix this issue of the confirmation of email because I was not able to confirm my email when I created my account in your application :fearful: It would be nice if you fix this issue of confirmation email :+1:

Hint: You can use a gem called [letter-opener](https://github.com/ryanb/letter_opener) for this purpose :blush:

Reply:
Confirmable was removed as it is not a required feature.

Reply:
I can understand @samlorlah this is not a requirement in your application but it is directly affecting your application. It is a general requirement that your application should work :blush: so if something is stopping your application to proceed then I would request the changes for that thing even if it is not the requirement. It is good that you have to remove it I won't stop you from doing it or I won't request changes for this if you remove it. But for the next project, you should understand that if something is stopping your application then we would request changes for that even if that feature is the requirement of not.

If you have any questions my dear brother then I am always in slack even I was in touch with your partner @shuhad786 so whenever you wanna discuss something then do reach out to me without any hesitation :+1: